src/term.rs

```diff
@@ -435,7 +435,7 @@ impl Term {
             Term::Record(..) | Term::RecRecord(..) => Some("Record"),
             Term::Array(..) => Some("Array"),
             Term::SealingKey(_) => Some("SealingKey"),
-            Term::Sealed(_, _) => Some("Sealed"),
+            Term::Sealed(_, _, _) => Some("Sealed"),
```

Comment:
Nitpicking, but using ellipsis for more than one element is more future proof (who knows, one day `Sealed` may become a 7-ary operator :sweat_smile: )

```suggestion
            Term::Sealed(..) => Some("Sealed"),
```